server/fixtures/article.js

```diff
@@ -4,7 +4,15 @@ const slugTitle = createUniqueSlug('hello world');
 
 export const newArticle = {
   title: 'Hello world!',
-  body: 'Just a dummy data',
+  body: `So how did the classical Latin become so incoherent? According to McClintock, a 15th century typesetter likely scrambled part of Cicero's De Finibus in order to provide placeholder text to mockup various fonts for a type specimen book.
```

Comment:
Is there a reason you decided to make the body this long?

Reply:
Yes, I want the body length to be greater than 200 so that createEllipsis function can be fully tested.
I will reduce it.